tests/tests.go

```diff
@@ -290,7 +290,12 @@ func initConfig(conf string, params ...helperParams) error {
 
 func execCmds(t *testing.T, cmds ...Cmd) (e error) {
 	for _, cmd := range cmds {
-		args := strings.Split(cmd.Cmd, " ")
+		// ugly workaround to handle arguments that contain spaces, these need to be escaped with \\
```

Comment:
what about doing something like this
https://play.golang.org/p/YRRnYC-UDqA

A kind of real command line parser

Reply:
You need somthing which mimic shell handling - for that you may consider:

https://github.com/kballard/go-shellquote